src/execution/index/art/art.cpp

```diff
@@ -804,7 +894,7 @@ bool ART::SearchCloseRange(ARTIndexScanState &state, ARTKey &lower_bound, ARTKey
 	return it.Scan(upper_bound, max_count, result_ids, right_equal);
 }
 
-bool ART::Scan(const Transaction &transaction, const DataTable &table, IndexScanState &state, const idx_t max_count,
+bool ART::Scan(const Transaction &transaction, const DataTable &table, IndexScanState &state, idx_t max_count,
```

Comment:
Why is this no longer `const`?

Reply:
It barely has any effect being const and I get CLion warnings by having it be const in the header as it has no effect there - but I can change it back.